CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog - v3
 
+## [v3.6.6] (Aug 3 2023)
+### Feat:
+* Add `customExtensionParams` for `sdk.addSendbirdExtensions` (#698)
+  The 3rd parameter customData to the `sdk.addSendbirdExtension` function, allowing it to be delivered from outside of UIKit React.
+  e.g.
+  ```
+  <SendbirdProvider
+    customExtensionParams={{
```

Comment:
minor suggestion, its better to memoize this object in this release note,
```
const memoizedCustomExtensionParams = useRef({ ... })
<SendbirdProvider
    customExtensionParams={memoizedCustomExtensionParams}
``` 
There is chance customers might shoot themselves in the foot later

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/699/commits/e245e7c3a92d8cd4eaeab5bd4c9aedc9d028e3b6 Yeah thanks for catching it out!